lib/feed.xml

```diff
@@ -53,26 +53,12 @@
       <content type="html" xml:base="{{ post.url | prepend: url_base | xml_escape }}">{{ post.content | strip | xml_escape }}</content>
 
       {% assign post_author = post.author | default: post.authors[0] | default: site.author %}
-      {% assign post_author_email = false %}
-      {% assign post_author_uri = false %}
+      {% assign post_author = site.data.authors[post_author] | default: post_author %}
```

Comment:
Wondering if, between this and Jekyll SEO Tag, it may make sense to have a shared `jekyll-author` gem, that would never be included directly (I *guess* it could?), but would handle author being a string, a hash, or a ref to `_data`, and would always return an object, maybe in Rubyland?